### PR TITLE
feat: add phygitals

### PR DESCRIPTION
**Pull request by albertfolch-redeemeum**
depends on: https://github.com/bosonprotocol/core-components/pull/673

also fixes: https://github.com/bosonprotocol/interface/issues/1077

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -197,6 +198,7 @@
     "istanbul": "^0.4.5",
     "lint-staged": "^13.0.3",
     "nyc": "^15.1.0",
+    "opensea-js": "^7.1.7",
```
Comment (albertfolch-redeemeum) [inline package.json:201]: TODO: remove

File: src/components/ui/Image.tsx [hunk 2]
```diff
@@ -115,7 +118,7 @@ const Image: React.FC<IImage & React.HTMLAttributes<HTMLDivElement>> = ({
   const isError = status === "error";
   const isLoading = status === "loading";
   const isSuccess = status === "success";
-
+  alt === "token ID 1" && console.log({ currentSrc, status, alt });
```
Comment (albertfolch-redeemeum) [inline src/components/ui/Image.tsx:121]: TODO: remove

File: src/lib/utils/hooks/useSwitchChain.ts [hunk 3]
```diff
@@ -21,7 +21,10 @@ function getRpcUrl(chainId: SupportedChainsType): string {
     case ChainId.MAINNET:
     case ChainId.GOERLI:
     case ChainId.SEPOLIA:
+    case 80002:
       return RPC_URLS[chainId][0];
+    case 31337:
+      throw new Error("chain 31337 is not supported");
```
[the remark this answers is not in the document]
Comment (levalleux-ludo) [reply src/lib/utils/hooks/useSwitchChain.ts:27]: It's a shame if we can't run the dApp on the local env.
The RPC URLS to be used on it is "`http://127.0.0.1:8545`" as defined in core-components: https://github.com/bosonprotocol/core-components/blob/7fecf7d25b38a71bb19dfa4455096b2065461cad/packages/common/src/configs.ts#L22

**Review by levalleux-ludo — APPROVED** [review]
Awesome work!

**Pull request merged**